earth_enterprise/BUILD_Ubuntu.md

```diff
@@ -39,7 +39,7 @@ sudo apt-get install \
     libx11-dev libxcursor-dev libxerces-c-dev libxft-dev libxinerama-dev \
     libxml2-dev libxml2-utils libxmu-dev libxrandr-dev libyaml-cpp-dev \
     openssl libpcre3 libpcre3-dev \
-    python-dev python-imaging python-psycopg2 python-setuptools python2.7 \
-    python2.7-dev python-lxml python-git \
+    python-dev python-imaging python-psycopg2 \
```

Comment:
I don't think you need to change this file to install python27-pip. I think Ubuntu works fine if you just install python-defusedxml.